code/src/index.css

```diff
@@ -11,3 +13,96 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.submitBox {
+background-color: rgb(242, 241, 241);
+}
+
+.submitForm {
+  display: flex;
+  flex-direction: column;
+  width: 95%;
+  margin: 0 auto;
+  }
+
+  .submitTitle {
+    font-size: 80%;
+    margin: 2%;
+  }
+
+  .inputArea {
+    height: 10%;
+    width: 95%;
+    margin: 2%;
+  }
+
+  .submitButton {
+    width: 60%;
+    border-radius: 25px;
+    padding: 2%;
+    margin: 2% 2% 5% 2%;
+    background-color: #FFADAD;
+  }
+
+  .submitButton:hover {
+    opacity: 0.5;
+  }
+
+.box {
+  font-family: "Roboto Mono";
+  width: 30%;
+  margin: 3% auto;
+  border: 1px solid black;
+  box-shadow: 5px 5px;
+}
+
+.boxCard {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.postedThought {
+  margin: 5%;
+  flex-basis: 100%;
```

Comment:
Really cool that you found/added this 🥇  and Thank you for taking the time to explain it during Team session.

When it comes to flexbox i make everything a "box" and then you can modify the length/size depening on what you want, nice to see a diffrent way